pkg/build/build.go

```diff
@@ -322,6 +323,15 @@ func WithDependencyLog(logFile string) Option {
 	}
 }
 
+// WithBinShOverlay sets a filename to copy from when installing /bin/sh
+// into a build environment.
+func WithBinShOverlay(binShOverlay string) Option {
+	return func(ctx *Context) error {
+		ctx.BinShOverlay = binShOverlay
```

Comment:
If this is a symlink does it still work?

Reply:
It does, because the data will get copied.